src/patternfly/components/Breadcrumb/breadcrumb-item.hbs

```diff
@@ -0,0 +1,9 @@
+<li class="pf-c-breadcrumb__item{{#if breadcrumb-item--modifier}} {{breadcrumb-item--modifier}}{{/if}}{{#if breadcrumb-item--current}} pf-m-active{{/if}}"
```

Comment:
Can you remove the conditional for active and just add it as a modifier in the example.